### DISCUSSION TITLE
remarkParse is treating indented blockQuote as code

### DISCUSSION THREAD

**Discussion by vritant24** [remarkjs/remark#1157] [Q&A] [answered]
I have the following mardkown I'm parsing:
```
# Heading
> Lorem ipsum dolor sit amet, consectetur adipisicing elit.
    
    > Lorem ipsum dolor sit amet, consectetur adipisicing elit.
```

When I process the markdown with the following: 
```
const testProcessor = unified()
            .use(remarkParse)
            .use(customPlugin)
            .use(remarkStringify);
```

in my plugin's `transform` method, I see the following tree provided:
![image](https://user-images.githubusercontent.com/13074644/236992216-670c288a-63af-4617-9ffb-e15a6e872ab6.png)

but when I change the markdown to the following (no extra line)
```
# Heading
> Lorem ipsum dolor sit amet, consectetur adipisicing elit.
    > Lorem ipsum dolor sit amet, consectetur adipisicing elit.
```

I see the following:
![image](https://user-images.githubusercontent.com/13074644/236992331-baea1ba5-4f36-41cf-8b87-b8e6c30d05dd.png)

I'm unsure where the extra code block is coming from and would appreciate any help in understanding it. 

I'm using the latest released versions of all packages here.

**Comment by wooorm** [comment, marked as answer]
This is how markdown works. https://spec.commonmark.org/0.30/#indented-code-blocks, https://spec.commonmark.org/0.30/#paragraphs.
It works the same here:

# Heading
> Lorem ipsum dolor sit amet, consectetur adipisicing elit.
    
    > Lorem ipsum dolor sit amet, consectetur adipisicing elit.

# Heading
> Lorem ipsum dolor sit amet, consectetur adipisicing elit.
    > Lorem ipsum dolor sit amet, consectetur adipisicing elit.

**Reply by vritant24** [reply]
Thanks for the detailed clarification, really appreciate it :) 

Was quite surprising to me as I hadn't run into it until now and assumed that only backticks created code blocks.

**Reply by wooorm** [reply]
Been there from the start of markdown! I like fenced code (first intro’d by GFM) much more though!
